ddp-workspace/projects/ddp-dsm-ui/src/app/FON/components/register-patients-modal/register-patients-modal.component.ts

```diff
@@ -44,11 +45,16 @@ export class RegisterPatientsModalComponent implements OnInit {
   }
 
   public registerPatient(): void {
-    const {patients} = this.addPatientForm.getRawValue();
-    this.closeDialog(patients);
+    console.log(this.addPatientForm.getRawValue());
```

Comment:
Do we need it?

Reply:
It's the only option we can see data flow at this moment. when backend is ready, I will remove that